include/grpc/grpc_security.h

```diff
@@ -834,30 +838,73 @@ typedef void (*grpc_tls_on_credential_reload_done_cb)(
     grpc_tls_credential_reload_arg* arg);
 
 /** A struct containing all information necessary to schedule/cancel a
-    credential reload request.
-    - cb and cb_user_data represent a gRPC-provided
-      callback and an argument passed to it.
-    - key_materials_config is an in/output parameter containing currently
-      used/newly reloaded credentials. If credential reload does not result in
-      a new credential, key_materials_config should not be modified. The same
-      key_materials_config object can be updated if new key materials is
-      available.
-    - status and error_details are used to hold information about
-      errors occurred when a credential reload request is scheduled/cancelled.
-    - config is a pointer to the unique grpc_tls_credential_reload_config
-      instance that this argument corresponds to.
-    - context is a pointer to a wrapped language implementation of this
-      grpc_tls_credential_reload_arg instance.
-    - destroy_context is a pointer to a caller-provided method that cleans
-      up any data associated with the context pointer.
+    credential reload request. To help understand each fields, we mark each
+    field with labels. Here are their explanations
+    -------------------------label explanation----------------------------------
+    [USER-MANAGED]: fields that are managed directly by the end users
```

Comment:
It's also not clear to me in what way the fields are managed by end users. Do we mean the ownership?

Reply:
I agree that "user-managed" is not very clear. Also does "user-managed" also mean user-supplied values? User-supplied values is important to convey (once "user-managed" is clarified).